src/emails/templates/monthlyActivity/freeUser/MonthlyActivityFreeUser.tsx

```diff
@@ -0,0 +1,290 @@
+  dataSummary: DashboardSummary;
+  unsubscribeLink: string;
+};
+
+export const MonthlyReportFreeUserEmail = (
+  props: MonthlyReportFreeUserEmailProps,
+) => {
+  const l10n = props.l10n;
+
+  const premiumSubscriptionUrlObject = new URL(
+    getPremiumSubscriptionUrl({ type: "yearly" }),
+  );
+  const assumedCountryCode = getSignupLocaleCountry(props.subscriber);
+  const hasRunFreeScan = props.subscriber.onerep_profile_id !== null;
+
+  const bannerDataCta = {
+    label: hasRunFreeScan
+      ? l10n.getString("email-monthly-report-free-banner-cta-upgrade")
+      : l10n.getString("email-monthly-report-free-banner-cta-free-scan"),
+    link: hasRunFreeScan
+      ? premiumSubscriptionUrlObject.href
+      : `${process.env.SERVER_URL}/user/dashboard/?utm_source=monitor-product&utm_medium=email&utm_campaign=${props.utmCampaignId}&utm_content=take-action${props.utmContentSuffix}`,
+  };
+
+  const purpleActiveColor = "#7542E5";
+  const greyInactiveColor = "#9E9E9E";
+
+  const resolvedBoxData = {
+    dataPointCountLabel: hasRunFreeScan
+      ? "email-monthly-report-free-summary-manually-resolved-exposures"
+      : "email-monthly-report-free-summary-resolved-breaches",
+    dataPointValue: hasRunFreeScan
+      ? props.dataSummary.dataBreachResolvedNum +
+        props.dataSummary.dataBrokerManuallyResolvedNum
+      : props.dataSummary.dataBreachResolvedNum,
+    activeState:
+      (hasRunFreeScan &&
+        props.dataSummary.dataBreachResolvedNum +
+          props.dataSummary.dataBrokerManuallyResolvedNum >
+          0) ||
+      (!hasRunFreeScan && props.dataSummary.dataBreachResolvedNum > 0),
+  };
+
+  const showCongratulatoryBanner = {
+    postScan:
+      hasRunFreeScan &&
+      props.dataSummary.dataBreachUnresolvedNum +
+        props.dataSummary.dataBrokerInProgressNum ===
+        0,
+    preScan: !hasRunFreeScan && props.dataSummary.dataBreachUnresolvedNum === 0,
+  };
+
+  return (
+    <mjml>
+      <mj-head>
+        <mj-preview>
+          {l10n.getString("email-monthly-report-hero-free-heading", {
+            monthOfReport: props.month,
+          })}
+        </mj-preview>
+        <mj-style>
+          {/* This class rounds the edges of a table element */}
+          {`
+            .stat_column table {
+                border-collapse: separate;
+                border-spacing: 0;
+                padding: 15px 0px;
+            }
+
+            .upgrade_link {
+              display: flex !important;
+              align-items: center;
+              gap: 10px;
+              justify-content: center;
+            }
+
+            .manually_resolved_column_sparkles {
+              background-image: url(${process.env.SERVER_URL}/images/email/monthly-activity/sparkles.png);
+              background-position: center;
+              background-size: 90%;
+              background-repeat: no-repeat;
+            }
+        `}
+        </mj-style>
+      </mj-head>
+      <mj-body>
+        <EmailHero
```

Comment:
Nit: The line height of the subheading is getting a bit narrow.
<img width="258" alt="image" src="https://github.com/user-attachments/assets/e6477b18-e8be-4d6e-9bcb-423d31c0f980">